flink-connector-oracle-cdc/src/test/java/com/ververica/cdc/connectors/oracle/table/OracleConnectorITCase.java

```diff
@@ -418,6 +418,77 @@ public void testConsumingNumericColumns() throws Exception {
+                Statement statement = connection.createStatement()) {
+            statement.execute(
+                    "CREATE TABLE debezium.xmltype_table ("
+                            + " ID NUMBER(4),"
+                            + " T15VARCHAR sys.xmltype,"
+                            + " PRIMARY KEY (ID))");
+            statement.execute(
+                    "INSERT INTO debezium.xmltype_table "
+                            + "VALUES (11, sys.xmlType.createXML('<name><a id=\"1\" value=\"some values\">test xmlType</a></name>'))");
+        }
+
+        String sourceDDL =
+                String.format(
+                        "CREATE TABLE test_xmltype_table ("
+                                + " ID INT,"
+                                + " T15VARCHAR STRING,"
+                                + " PRIMARY KEY (ID) NOT ENFORCED"
+                                + ") WITH ("
+                                + " 'connector' = 'oracle-cdc',"
+                                + " 'hostname' = '%s',"
+                                + " 'port' = '%s',"
+                                + " 'username' = '%s',"
+                                + " 'password' = '%s',"
+                                + " 'database-name' = 'XE',"
+                                + " 'schema-name' = '%s',"
+                                + " 'table-name' = '%s'"
+                                + ")",
+                        oracleContainer.getHost(),
+                        oracleContainer.getOraclePort(),
+                        "dbzuser",
+                        "dbz",
+                        "debezium",
+                        "xmltype_table");
+        String sinkDDL =
+                "CREATE TABLE test_xmltype_sink ("
+                        + " id INT,"
+                        + " T15VARCHAR STRING,"
+                        + " PRIMARY KEY (id) NOT ENFORCED"
+                        + ") WITH ("
+                        + " 'connector' = 'values',"
+                        + " 'sink-insert-only' = 'false',"
+                        + " 'sink-expected-messages-num' = '1'"
+                        + ")";
+        tEnv.executeSql(sourceDDL);
+        tEnv.executeSql(sinkDDL);
+
+        // async submit job
+        TableResult result =
+                tEnv.executeSql("INSERT INTO test_xmltype_sink SELECT * FROM test_xmltype_table");
+
+        waitForSnapshotStarted("test_xmltype_sink");
+
+        // waiting for change events finished.
+        waitForSinkSize("test_xmltype_sink", 1);
+
+        List<String> expected =
+                Arrays.asList(
+                        "+I[11, <name>\r\n"
+                                + "   <a id=\"1\" value=\"some values\">test xmlType</a>\r\n"
+                                + "</name>]");
+
+        List<String> actual = TestValuesTableFactory.getRawResults("test_xmltype_sink");
+        Collections.sort(actual);
+        assertEquals(expected, actual);
```

Comment:
The assertion failed in CI, could you fix the test, I guess the failure reason may relate to the expected string contains some special char.

```
ul 05 06:23:31 06:23:31.594 [ERROR] Tests run: 5, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 209.833 s <<< FAILURE! - in com.ververica.cdc.connectors.oracle.table.OracleConnectorITCase
Jul 05 06:23:31 06:23:31.594 [ERROR] testXmlType(com.ververica.cdc.connectors.oracle.table.OracleConnectorITCase)  Time elapsed: 22.203 s  <<< FAILURE!
Jul 05 06:23:31 java.lang.AssertionError: 
Jul 05 06:23:31 expected:<[+I[11, <name>
Jul 05 06:23:31    <a id="1" value="some values">test xmlType</a>
Jul 05 06:23:31 </name>]]> but was:<[+I[11, <name>
Jul 05 06:23:31    <a id="1" value="some values">test xmlType</a>
Jul 05 06:23:31 </name>]]>
Jul 05 06:23:31 	at com.ververica.cdc.connectors.oracle.table.OracleConnectorITCase.testXmlType(OracleConnectorITCase.java:488)

```

Reply:
My local(Windows) test can pass,show the expected line spearator is `CRLF`, I'll try to fix it.